src/XMonad/Operations.hs

```diff
@@ -190,8 +191,9 @@ windowBracket p action = withWindowSet $ \old -> do
     windows         $ \_ -> new
   return a
 
--- | A version of @windowBracket@ that discards the return value, and handles an
--- @X@ action reporting its need for refresh via @Any@.
+-- | Perform an @X@ action. If it returns @Any True@, unwind the changes to the @WindowSet@ and replay them using @windows@.
+-- This is a version of @windowBracket@ that discards the return value and
+-- handles an @X@ action that reports its need for refresh via @Any@.
```

Comment:
I think it's worth trying to be consistent with the doc-style of the rest of the module here (and also in other parts); meaning line-breaks after what I assume are 71 or 79 columns.